src/bokeh/plotting/glyph_api.py

```diff
@@ -1115,7 +1115,8 @@ def scatter(self, *args: Any, **kwargs: Any) -> GlyphRenderer:
 
             size (str or list[float]) : values or field names of sizes in |screen units|
 
-            marker (str, or list[str]): values or field names of marker types
+            marker (str, or list[str]): values or field names of marker types, visit
+                :class:`~bokeh.core.enums.MarkerType` for a complete list of valid values.
```

Comment:
Perhaps this could use an update after PR #14165.

Reply:
Comment to myself: https://github.com/bokeh/bokeh/blob/branch-3.7/docs/bokeh/source/docs/user_guide/basic/scatters.rst needs an update, too.